libgalois/include/galois/LargeVector.h

```diff
@@ -68,9 +68,9 @@ class LargeVector : public boost::noncopyable {
     std::cout << "new_cap = " << new_cap << "\tmmap_size = " << mmap_size
               << std::endl;
 
-    m_data =
-        static_cast<T*>(mmap(nullptr, mmap_size, PROT_READ | PROT_WRITE,
-                             MAP_SHARED | MAP_HUGETLB | MAP_HUGE_2MB, m_fd, 0));
+    m_data = static_cast<T*>(
+        mmap(nullptr, mmap_size, PROT_READ | PROT_WRITE,
+             MAP_SHARED | MAP_HUGETLB | MAP_HUGE_2MB | MAP_POPULATE, m_fd, 0));
```

Comment:
Yes

Reply:
this turns out to not improve performance (though it does not change scaling).

Reply:
as in no change?